lib/srv/sess.go

```diff
@@ -1272,34 +1592,36 @@ func (s *session) getParties() (parties []*party) {
 	return parties
 }
 
-func newMultiWriter() *multiWriter {
-	return &multiWriter{writers: make(map[string]writerWrapper)}
+func NewMultiWriter() *MultiWriter {
+	return &MultiWriter{writers: make(map[string]writerWrapper)}
 }
 
-type multiWriter struct {
+type MultiWriter struct {
 	mu           sync.RWMutex
 	writers      map[string]writerWrapper
 	recentWrites [][]byte
+	OnError      func(string, error)
 }
 
```

Comment:
A lot of the code here looks like a re-implementation of lib/srv/termmanager.go. Is that true or am I confused?

Reply:
It was marked as in-use by VSCode and I didn't think much of it but it was marked unused when I restarted the Go LSP.